library/Zend/Form/Element/DateSelect.php

```diff
@@ -89,7 +89,7 @@ public function getDayAttributes()
     /**
      * @param  string|array|\ArrayAccess|PhpDateTime $value
      * @throws \Zend\Form\Exception\InvalidArgumentException
-     * @return void|\Zend\Form\Element
+     * @return DateSelect
```

Comment:
change only for this method i think because it is new introduced docblock. there is inconsistencies about it in other files too. i created PR in the past but can't finish to update all.